bucket/clash-for-windows.json

```diff
@@ -0,0 +1,29 @@
+{
+    "homepage": "https://github.com/Fndroid/clash_for_windows_pkg",
+    "description": "A Windows/macOS GUI based on Clash",
```

Comment:
- MacOS is irrelevant
- what is clash?

Reply:
Clash is a rule-based tunnel in Go according to the clash introduction, and it doesnot have a gui. https://github.com/Dreamacro/clash
I'll change the description later.